content/powered-gvisor.png

```diff
@@ -0,0 +1 @@
+../assets/logos/powered-gvisor.png
```

Comment:
You shouldn't need a symbolic link into the content directory.

Make a folder static/img and put the powered-gvisor.png in there. The url would then be /img/powered-gvisor.png